source/Ship.cpp

```diff
@@ -2662,7 +2662,10 @@ int Ship::RequiredCrew() const
 
 int Ship::CrewValue() const
 {
-	return max(Crew(), RequiredCrew()) + attributes.Get("crew equivalent");
+	int crewEquivalent = attributes.Get("crew equivalent");
+	if(attributes.Get("crew equivalent overrides"))
```

Comment:
This name more clearly explains what the attribute actually does. Thoughts?
```suggestion
	if(attributes.Get("crew equivalent as crew"))
```

Reply:
Its a bit long and the reputation factor isnt in it
"crew equivalent reputation", maybe? Idk

Reply:
It's long, but it's also not something we should be using in a place the player could see anyway.